package.json

```diff
@@ -54,25 +56,26 @@
     "node-emoji": "^1.7.0",
     "parse-diff": "^0.4.0",
     "query-string": "^4.3.1",
-    "react": "16.0.0-alpha.12",
-    "react-native": "0.48.1",
+    "react": "^16.0.0",
+    "react-native": "^0.48.4",
     "react-native-actionsheet": "^2.2.0",
     "react-native-app-intro": "^1.1.5",
     "react-native-code-push": "^5.0.0-beta",
     "react-native-communications": "^2.2.1",
     "react-native-config": "^0.6.0",
     "react-native-cookies": "^3.2.0",
     "react-native-device-info": "^0.11.0",
-    "react-native-elements": "^0.9.3",
+    "react-native-elements": "^0.17.0",
     "react-native-htmlview": "^0.12.0",
     "react-native-i18n": "^2.0.4",
     "react-native-material-design-searchbar": "^1.1.4",
-    "react-native-parallax-scroll-view": "^0.19.0",
+    "react-native-mock": "https://github.com/shqld/react-native-mock/tarball/master",
+    "react-native-parallax-scroll-view": "https://github.com/mini-eggs/react-native-parallax-scroll-view/tarball/react16-support",
```

Comment:
Definitely need to wait until these are all actually fixed before merging this. Using these forks is just to get it to work, not a permanent (or acceptable) solution.